wremnants/include/recoil_helper.h

```diff
@@ -149,8 +149,8 @@ class RecoilCalibrationHelper {
         Eigen::TensorFixedSize<double, Eigen::Sizes<>> ut_para_tensor;
         Eigen::TensorFixedSize<double, Eigen::Sizes<>> ut_perp_tensor;
         pt_tensor(0) = pt;
-        ut_para_tensor(0) = ut_para;
-        ut_perp_tensor(0) = ut_perp;
+        ut_para_tensor(0) = std::isfinite(ut_para) ? ut_para : 13000.0;
```

Comment:
I think these changes are no longer necessary, it was a fix I had come up with before jan fixed it in a different way.

Reply:
In this case I suggest to revert the changes here. It could be dangerous to just set Nans to 13000 if we don't understand the cause of it.